src/licensedcode/data/licenses/Scilab-old.LICENSE

```diff
@@ -0,0 +1,289 @@
+
+   3. The INRIA and the ENPC authorize you free of charge to circulate
+      and distribute for no charge, for non-commercial purposes the
+      source and/or object code of DERIVED SOFTWARE on any present and
+      future support, providing: * the reference " Scilab inside
+      (c)INRIA-ENPC " is prominently mentioned; * the DERIVED SOFTWARE
+      is distributed under the present license; * the recipients of
+      the distribution can access the SOFTWARE code source; * the
+      DERIVED SOFTWARE is distributed under a name other than SCILAB.
+
+   4. Any commercial use or circulation of the DERIVED SOFTWARE shall
+      have been previously authorized by INRIA and ENPC.
+
+5- Object and conditions of the license concerning COMPOSITE SOFTWARE
+
+   1. INRIA and ENPC authorize you to reproduce and interface all or
+      part of the SOFTWARE with all or part of other software,
+      application packages or toolboxes of which you are owner or
+      entitled beneficiary in order to obtain COMPOSITE SOFTWARE.
+
+   2. INRIA and ENPC authorize you free, of charge, to use the
+      SOFTWARE source and/or object code included in the COMPOSITE
+      SOFTWARE, without restriction, providing the following statement
+      appears in all the copies: "composite software using Scilab
+      (c)INRIA-ENPC functionality".
+
+   3. INRIA and ENPC authorize you, free of charge, to circulate and
+      distribute for no charge, for purposes other than commercial,
+      the source and/or object code of COMPOSITE SOFTWARE on any
+      present and future support, providing: * the following reference
+      is prominently mentioned: "composite software using Scilab
+      (c)INRIA-ENPC functionality "; * the SOFTWARE included in
+      COMPOSITE SOFTWARE is distributed under the present license ; *
+      recipients of the distribution have access to the SOFTWARE
+      source code; * the COMPOSITE SOFTWARE is distributed under a
+      name other than SCILAB.
+
+   4. Any commercial use or distribution of COMPOSITE SOFTWARE shall
+      have been previously authorized by INRIA and ENPC.
+
+6- Limitation of the warranty
+
+Except when mentioned otherwise in writing, the SOFTWARE is supplied
+as is, with no explicit or implicit warranty, including warranties of
+commercialization or adaptation. You assume all risks concerning the
+quality or the effects of the SOFTWARE and its use. If the SOFTWARE is
+defective, you will bear the costs of all required services,
+corrections or repairs.
+
+7- Consent
+
+When you access and use the SOFTWARE, you are presumed to be aware of
+and to have accepted all the rights and obligations of the present
+license.
+
+8- Binding effect
+
+This license has the binding value of a contract.
+
+You are not responsible for respect of the license by a third party.
+
+9- Applicable law
+
+The present license and its effects are subject to French law and the
+competent French courts.
+
+
+
+
+
+FR:
```

Comment:
We avoid keeping two text in different languages in the same files. These would be two different licenses instead.

Reply:
Do you mean I should create two licenses for french and english (something like <code>scilab-en</code> and <code> scilab-fr</code>)?

Reply:
yes